### PR TITLE
Insteon change backend module to pyinsteon from insteonplm

### PR DESCRIPTION
**Pull request by teharris1**
## Proposed change

Update documentation to reflect the changes when migrating the backend module from `insteonplm` to `pyinsteon`.

## Type of change

- [  ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/35198
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/insteon.markdown [hunk 1]
```diff
@@ -81,101 +81,86 @@ insteon:
        unitcode: UNITCODE
        platform: PLATFORM
        steps: STEPS
-  x10_all_units_off: HOUSECODE
-  x10_all_lights_on: HOUSECODE
-  x10_all_lights_off: HOUSECODE
 ```
 
 {% configuration %}
```
Comment (klaasnicolaas) [inline source/_integrations/insteon.markdown:86]: Please revert the style format from here

File: source/_integrations/insteon.markdown [hunk 2]
```diff
@@ -81,101 +81,86 @@ insteon:
...
+type: integer
+platform:
+description: "is the Home Assistant Platform to associate the device with. The following platforms are supported: binary_sensor: Used for on/off devices or keypad buttons that are read-only. light: Used for dimmable X10 devices. switch: Used for On/Off X10 devices."
+required: true
+type: string
+dim_steps:
+description: is the number of dim/bright steps the device supports. Used for dimmable X10 devices only.
+required: false
+default: 22
+type: integer
```
Comment (klaasnicolaas) [inline source/_integrations/insteon.markdown]: Until here

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/insteon.markdown (hunk 2 above)
Comment (teharris1) [reply source/_integrations/insteon.markdown]: @klaasnicolaas I made the change above but I am a bit confused. When I look at the changes they appear correct in the editor but they don't appear to be correct after saving. Can you have a look and tell me if they are correct?

**Comment by teharris1** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.
> (message by DocsTargetBranch)

This is not a PR against the current functionality. It is to support a code PR #13296. I believe it is correct to have this against `next` not `current`.

**Comment by klaasnicolaas** [comment]
@teharris1 You are getting this message from probot because you didn't provide the link to the parent PR in the beginning, so probot thinks you did something wrong.

**Comment by teharris1** [comment]
> ## Additional information
> * Link to parent pull request in the codebase: [home-assistant/core#35198](https://github.com/home-assistant/core/pull/35198)

I thought I did. What did I do wrong? Do I need to fix that?

**Review by frenck — APPROVED** [review]
Thanks, @teharris1 👍

**Pull request merged**